pkg/controller/syncset/syncset_controller.go

```diff
@@ -113,7 +113,7 @@ func (r *ReconcileSyncSet) Reconcile(ctx context.Context, request reconcile.Requ
 			return reconcile.Result{}, err
 		}
 	}
-	sk := aggregator.Key{Source: "syncset", ID: request.NamespacedName.String()}
+	sk := aggregator.Key{Source: "syncset", ID: request.String()}
```

Comment:
request.String() returns alot more things than just namespace/name. can you pls just use that?

Reply:
`NamespacedName` is embedded in `request` so `request.String() = request.NamespacedName.string()` here.